tasks/test.py

```diff
@@ -157,8 +157,19 @@ def lint_releasenote(ctx):
             return
 
         # Then check that at least one note was touched by the PR
-        res = requests.get("https://api.github.com/repos/DataDog/datadog-agent/pulls/{}/files".format(pr_id))
-        files = res.json()
+        filenames = []
+        url = requests.get("https://api.github.com/repos/DataDog/datadog-agent/pulls/{}/files".format(pr_id))
+        # traverse paginated github response
+        while True:
+            res = requests.get(url)
+            for f in res.json():
+                filenames.append(f['filename'])
```

Comment:
We could directly loop through the file here and break if we found a release note. and add the raise in an `else` statement.